ArcGISRuntimeSDKQt_CppSamples/Scenes/SyncMapViewSceneView/README.metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "category": "Scenes",
-    "description": "Keep the viewpoints of two views (e.g. MapView and SceneView) synchronized with each other.",
+    "description": "Keep the view points of two views (e.g. MapView and SceneView) synchronized with each other.",
```

Comment:
I believe it was one word before since a "viewpoint" is a specific thing in the Runtime API

Reply:
```suggestion
    "description": "Keep the viewpoints of two views (e.g. MapView and SceneView) synchronized with each other.",
```

Reply:
That's what I thought too, so I reverted that change and updated the source README